app/javascript/react/components/BrandTile.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const BrandTile = (props) => {
+
+	return(
+		<div className='cell small-4 pizzaCard'>
+			<div className='card'>
+				<div className='card-section text-center'>
+					<Link to={`/pizzas/${props.id}`}>{props.name}</Link>
+				</div>
+			</div>
+		</div>
+	)
+}
+
```

Comment:
indentation is WACK in this file. please adjust! this may be due to someones editor setting. Checkout the staff announcement Kerrin made last week regarding this

Reply:
Ill allow this for the sake of moving on though